src/cpp/r/R/Api.R

```diff
@@ -672,6 +653,26 @@
                                             line = -1L, 
                                             col = -1L, 
                                             moveCursor = TRUE) {
+
+   # validate path argument
+   hasFile <- !is.null(path) && length(path) > 0
+   if (hasFile && !is.character(path)) {
+      stop("path must be a character")
+   }
+   if (hasFile && !file.exists(path)) {
+      stop(path, " does not exist.")
+   }
+   
+   if (hasFile)
+   {
+      # expand and alias for client
+      path <- .rs.normalizePath(path, winslash = "/", mustWork = TRUE)
+      homeDir <- path.expand("~")
```

Comment:
Maybe in addition, because: 

``` r
.rs.createAliasedPath("/Users/romainfrancois/git/tidyverse/dplyr/R/across.R")
#> [1] "~/git/tidyverse/dplyr/R/across.R"
.rs.createAliasedPath("~/git/tidyverse/dplyr/R/across.R")
#> [1] "~/git/tidyverse/dplyr/R/across.R"
.rs.createAliasedPath("R/across.R")
#> [1] "R/across.R"
```

Without this pull request, when we do `.rs.api.documentOpen("R/across.R")` and the file was already open otherwise (with the full path `~/git/tidyverse/dplyr/R/across.R`) it opens it again. 

<img width="301" alt="image" src="https://user-images.githubusercontent.com/2625526/153578443-1b1e4bde-c945-4870-bcce-748454b39882.png">